arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+          "description": "The location of the workspace."
+        }
+      },
+      "description": "Metadata for a workspace that isn't linked to an Azure subscription."
+    },
+    "Tag": {
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The tag name."
+        },
+        "Value": {
+          "type": "string",
+          "description": "The tag value."
+        }
+      },
+      "required": [
+        "Name",
+        "Value"
+      ],
+      "description": "A tag of a saved search."
+    },
+    "SavedSearchProperties": {
+      "properties": {
+        "Category": {
+          "type": "string",
+          "description": "The category of the saved search. This helps the user to find a saved search faster. "
+        },
+        "DisplayName": {
+          "type": "string",
+          "description": "Saved search display name."
+        },
+        "Query": {
+          "type": "string",
+          "description": "The query expression for the saved search. Please see https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-search-reference for reference."
+        },
+        "Version": {
+          "type": "integer",
+          "format": "int64",
+          "minimum": 1,
+          "maximum": 1,
+          "description": "The version number of the query lanuage. Only verion 1 is allowed here."
+        },
+        "Tags": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Tag"
+          },
+          "description": "The tags attached to the saved search."
+        }
+      },
+      "required":[
+        "Category",
+        "DisplayName",
+        "Query",
+        "Version"
+      ],
+      "description": "Value object for saved search results."
+    },
+    "SavedSearchCreateOrUpdateParameters": {
+      "properties": {
+        "etag": {
+          "type": "string",
+          "description": "The etag of the saved search."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SavedSearchProperties",
+          "description": "Gets or sets properties of the saved search."
+        }
+      },
+      "description": "Parameters for creating/updating a saved search"
+    },
+    "CoreSummary": {
+      "properties": {
+        "Status": {
+          "type": "string",
+          "description": "The status of a core summary."
+        },
+        "NumberOfDocuments": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of documents of a core summary."
+        }
+      },
+      "required": [
+        "NumberOfDocuments" 
+      ],
+      "description": "The core summary of a search."
+    },
+    "SearchSort": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the field the search query is sorted on."
+        },
+        "order": {
+          "type": "string",
```

Comment:
can this be an enum? sounds like the possible values are something like "ascending"/"descending". Otherwise, describe what a user can expect here (if I see correctly, this is only part of responses, so it's not critical but still useful for users who want to use this value).